apps/emqx_rule_engine/src/emqx_rule_engine.erl

```diff
@@ -95,6 +95,9 @@
             end
         end()).
 
+-define(GET_RES_ALIVE_TIMEOUT, 60000).
+-define(TEST_RES_PREFIX, "_@test@_:").
```

Comment:
Test-> probe
And maybe avoid using @ in the id, not nice for log indexer.
_ instead?

Reply:
changed